tests/test_settings.py

```diff
@@ -816,6 +817,30 @@ class Config:
     assert Settings().dict() == {'foo': 'foo_secret_value_str'}
 
 
+@pytest.mark.skipif(sys.platform != 'linux', reason='Linux tests')
```

Comment:
do we actually need this? Does it fail on mac or windows?

Reply:
Yes, we need this because the issue exist only on Linux.

File names with capital letters are returned on macos and windows

Reply:
I have removed the skip